site/content/3.13/graphs/graph-analytics.md

```diff
@@ -81,8 +125,56 @@ setting in ArangoGraph:
   These session tokens need to be renewed every hour by default. See
   [HTTP API Authentication](../develop/http-api/authentication.md#jwt-user-tokens)
   for details.
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Start and stop Graph Analytics Engines
+
+The interface for managing the engines depends on the environment you use:
+
+- **ArangoDB Platform**: [GenAI service](#genai-service)
+- **ArangoGraph**: [Management API](#management-api)
+
+### GenAI service
+
+{{< tag "ArangoDB Platform" >}}
+
+GAEs are deployed and deleted via the [GenAI service](../data-science/graphrag/services/gen-ai.md)
+in the ArangoDB Platform.
+
+If you use cURL, you need to use the `-k` / `--insecure` option for requests
+if the Platform deployment uses a self-signed certificate (default).
+
+#### Deploy
+
+```sh
+#$JWT="<token>" # TODO: Rename from JWT to token? Can be access token or session token, perhaps also superuser token (and HTTP Basic auth should work too)
+Service=$(curl -sSk -H "Authorization: bearer $JWT" -XPOST https://127.0.0.1:8529/gen-ai/v1/graphanalytics)
+$ServiceID = $(echo $Service | jq -r .serviceInfo.serviceId)
+if [[ "$ServiceID" == "null" ]]; then 
+  echo "Error starting gral engine"
+else
+  echo "Engine started successfully"
+fi
+echo "$Service" | jq
+```
+
+#### List
+
+```sh
+curl -sSk -H "Authorization: bearer $JWT" -XPOST https://127.0.0.1:8529/gen-ai/v1/list_services | jq
+```
+
+#### Delete
+
+```sh
+curl -sSk -H "Authorization: bearer $JWT" -XDELETE https://127.0.0.1:8529/gen-ai/v1/service/$ServiceID | jq
+```
+
+### Management API
```

Comment:
We should add an introductory sentence here to be consistent with the other section (a short overview to introduce what you can do via the Management API).